File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/Form/Programming/OrientationChangedPage.cs
using System;
using System.Text;
using System.Data;
using System.Drawing;
using System.ComponentModel;
using System.Collections.Generic;

using Gizmox.WebGUI.Forms;
using Gizmox.WebGUI.Common;

namespace CompanionKit.Controls.Form.Programming
{
    public partial class OrientationChangedPage : UserControl
    {
        public OrientationChangedPage()
        {
            InitializeComponent();
            
        }

        /// <summary>
        /// Handles the Click event of the mobjOpenButton control.
        /// </summary>
        /// <param name="sender">The source of the event.</param>
        /// <param name="e">The <see cref="EventArgs"/> instance containing the event data.</param>
        private void mobjOpenButton_Click(object sender, EventArgs e)
        {
            //Shows OrientationForm instance
            OrientationForm mobjForm = new OrientationForm();
            mobjForm.WindowState = FormWindowState.Maximized;
            mobjForm.Show();
        }


    }
}

File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/Form/Programming/OrientationChangedPage.vb
Namespace CompanionKit.Controls.Form.Programming

	Public Class OrientationChangedPage

		Public Sub New()

			' This call is required by the Visual WebGui UserControl Designer.
			InitializeComponent()

	        

		End Sub

        ''' <summary>
        ''' Handles the Click event of the mobjOpenButton control.
        ''' </summary>
        ''' <param name="sender">The source of the event.</param>
        ''' <param name="e">The <see cref="System.EventArgs"/> instance containing the event data.</param>
        Private Sub mobjOpenButton_Click(sender As System.Object, e As System.EventArgs) Handles mobjOpenButton.Click
            'Shows OrientationForm instance
            Dim mobjForm As New OrientationForm()
            mobjForm.WindowState = FormWindowState.Maximized
            mobjForm.Show()
        End Sub
    End Class

End Namespace